bundles/core/src/test/resources/list/v3/exporter-childrenListType.json

```diff
@@ -6,61 +6,65 @@
             "id": "list-f56abdba21-item-70236eb2fc",
             "path": "/content/list/pages/page_1/page_1_1",
             "link": {
-                "url": "/content/list/pages/page_1/page_1_1.html",
-                "valid": true
+                "valid": true,
+                "url": "/content/list/pages/page_1/page_1_1.html"
             },
+            "description": "Description for Page 1.1",
             "title": "Page 1.1",
+            ":type": "cq:PageContent",
             "dataLayer": {
                 "list-f56abdba21-item-70236eb2fc": {
-                    "@type": "cq:PageContent",
+                    "@type": "cq:Component/item",
```

Comment:
This is weird, isn't it supposed to still be `cq:PageContent`, like the `:type`?

Reply:
This doesn't seem to be related to the changes in this PR. I created https://github.com/adobe/aem-core-wcm-components/issues/1589 to tackle it.